observability/build.gradle

```diff
@@ -8,12 +8,31 @@ plugins {
 
 description = "gRPC: Observability"
 dependencies {
-    api project(':grpc-api')
+    def cloudLoggingVersion = '3.6.1'
 
-    implementation libraries.guava,
-            project(':grpc-protobuf'),
+    api (project(':grpc-api')) {
+        exclude group: 'com.google.guava', module: 'guava'
+    }
+
+    api (project(':grpc-alts')) {
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'com.google.auth', module: 'google-auth-library-oauth2-http'
+    }
+
+    api 'com.google.guava:guava:31.0.1-jre',
```

Comment:
I understand this is derived from the quick test I did and our internal discussion. However we might not need `api` dependency but just `implementation` dependency might be sufficient (at least on some of them). Pls update after verifying.